org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/preferences/fields/AbstractDetailsPart.java

```diff
@@ -104,7 +104,6 @@ public final void propertyChange(PropertyChangeEvent event) {
 		if (event.getSource() instanceof FieldEditor) {
 			FieldEditor fe = (FieldEditor) event.getSource();
 			internalStore.setValue(fe.getPreferenceName(), fe.getPreferenceStore().getDefaultString(fe.getPreferenceName()));
-			fe.setPreferenceStore(internalStore);
```

Comment:
wonder why this was here in the first place

Reply:
this changes seems to break the editor changing on preference change

Reply:
I was to quick. I'll have another look.

Reply:
looks better now. lets see what the build tells